### DISCUSSION TITLE
Mark unexpected_successes (u) as failure

### DISCUSSION THREAD

**Discussion by wyz23x2** [CleanCut/green#240] [Enhancements]
The name states the problem. It's ``unexpected``, meaning that's the wrong behavior. Maybe it could change.  
Before:  
``OK (passes=4, unexpected_successes=1)``  
After:  
``FAILURE (passes=4, unexpected_successes=1)``

**Comment by CleanCut** [comment]
I have to admit, I've never encountered an unexpected success in real life. 😆  Sounds like a bug!  (And logged as https://github.com/CleanCut/green/issues/241)

**Reply by CleanCut** [reply]
Fix included in 3.2.6, just released.
